environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -994,6 +1089,8 @@ objects:
           - --whitelist={__name__="node_role_os_version_machine:cpu_capacity_cores:sum"}
           - --whitelist={__name__="node_role_os_version_machine:cpu_capacity_sockets:sum"}
           - --whitelist={__name__="subscription_sync_total"}
+          - --whitelist={__name__="csv_succeeded"}
```

Comment:
Is this change related?

Reply:
This is here because I have also updated dependencies, which comes from https://github.com/openshift/telemeter/pull/253/files. They should've created a PR for here as well.

Should I exclude this change and create another PR for it?

Reply:
No it's fine, I think thanks for context!